### PR TITLE
Info command, API improvements, New placeholder format

### PR DESCRIPTION
**Pull request by dlsf**
Pretty self-explanatory, please don't squash-merge if possible because these are almost independent commits.

### CODE REVIEW

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/utils/Placeholder.java [hunk 1]
```diff
@@ -18,7 +18,7 @@
      * @param value The actual value of the placeholder
      */
     public Placeholder(String key, String value) {
-        this.key = "{" + key + "}";
+        this.key = "%" + key + "%";
```
Comment (PeachesMLG) [inline src/main/java/com/iridium/iridiumskyblock/utils/Placeholder.java:21]: Need to change the placeholders in inventories.json and other configs to match this

**Review by PeachesMLG — CHANGES_REQUESTED** [review]

File: src/main/java/com/iridium/iridiumskyblock/commands/InfoCommand.java [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+     *
+     * @param commandSender The CommandSender which tries to tab-complete
+     * @param command       The command
+     * @param label         The label of the command
+     * @param args          The arguments already provided by the sender
+     * @return The list of tab completions for this command
+     */
+    @Override
+    public List<String> onTabComplete(CommandSender commandSender, org.bukkit.command.Command command, String label, String[] args) {
+        if (commandSender instanceof Player) {
```
[the remark this answers is not in the document]
Comment (dlsf) [reply src/main/java/com/iridium/iridiumskyblock/commands/InfoCommand.java:111]: uhm

[the remark this answers is not in the document]
Comment (PeachesMLG) [reply src/main/java/com/iridium/iridiumskyblock/commands/InfoCommand.java:111]: returning null auto does Player complete

**Comment by dlsf** [comment]
I hate Git so much, why did it create 4 new commits

**Pull request merged**